### PR TITLE
More README updates

### PR DESCRIPTION
**Pull request by retlehs**
WIP

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,227 +43,74 @@ Example `Vagrantfile` for Windows can be found [here](https://gist.github.com/st
...
+  * `db_user` - database username
+  * `db_password` - database password
+  * `db_host` - database hostname
+
+Additional options:
+
+* `admin_password` - WP admin user password
+* `admin_email` - WP admin email address
+* `user` - user owner of site directories/files
+* `group` - group owner of site directories/files
```
Comment (swalkinshaw) [inline README.md]: user and group are gone

File: README.md [hunk 2]
```diff
@@ -76,227 +43,74 @@ Example `Vagrantfile` for Windows can be found [here](https://gist.github.com/st
...
+* `site_install` - whether to install WordPress or not
+* `site_title` - WP site title
+* `db_create` - whether to auto create a database or not
+* `db_import` - Path to local `sql` dump file which will be imported
+* `system_cron` - Disable WP cron and use system's
+* `run_composer` - Run `composer install` before WP install
```
Comment (swalkinshaw) [inline README.md]: Gone as well

File: README.md [hunk 3]
```diff
@@ -2,52 +2,30 @@
 
 [![Build Status](https://travis-ci.org/roots/bedrock-ansible.svg)](https://travis-ci.org/roots/bedrock-ansible)
 
-bedrock-ansible is a set of [Ansible](http://www.ansible.com/home) [playbooks](http://docs.ansible.com/playbooks.html) designed to be used with [Bedrock](http://roots.io/wordpress-stack/) to configure development & production servers for Bedrock-based WordPress sites.
+Easily create development environments with Vagrant to help achieve development & production parity. Automatically configure servers and deploy WordPress sites.
 
-This project will let you configure complete WordPress ready servers with a single command:
+bedrock-ansible is a set of [Ansible](http://www.ansible.com/home) [playbooks](http://docs.ansible.com/playbooks.html). Configure complete [Bedrock](https://roots.io/bedrock/)-based WordPress ready servers with a single command:
```
Comment (swalkinshaw) [inline README.md]: Opening paragraphs are a little jumbled.

Maybe:

> bedrock-ansible is a set of Ansible playbooks to automatically configure servers and deploy WordPress sites. It easily creates development environments with Vagrant to help achieve development & production parity.

> Configure complete Bedrock-based WordPress ready servers with a single command:

File: README.md [hunk 4]
```diff
@@ -76,227 +43,71 @@ Example `Vagrantfile` for Windows can be found [here](https://gist.github.com/st
...
-
-From http://ferm.foo-projects.org/
-
-You can read the role documentation for `fail2ban` [here](roles/fail2ban/README.md) and `ferm` [here](roles/ferm/README.md).
-
-## Todo
-
-* Multisite: basic support is included but not yet complete. There are issues with doing a network install from scratch via WP-CLI.
-* Nginx: configuration needs more options and advanced setups like static files and subdomain multisite support.
+* [Locking down root](https://github.com/roots/bedrock-ansible/wiki/Security#locking-down-root)
```
Comment (swalkinshaw) [inline README.md]: Should just expand this a little:

> The `secure-root.yml` playbook is provided to help secure your remote servers including better SSH security.
   See the Wiki for [Locking down root](https://github.com/roots/bedrock-ansible/wiki/Security#locking-down-root).

**Comment by retlehs** [comment]
squashed

**Comment by retlehs** [comment]
https://github.com/roots/bedrock-ansible/wiki/Vagrant-box
https://github.com/roots/bedrock-ansible/wiki/Windows
https://github.com/roots/bedrock-ansible/wiki/Passwords
https://github.com/roots/bedrock-ansible/wiki/Vagrantfile
https://github.com/roots/bedrock-ansible/wiki/Mail

**Pull request merged**